core/common/src/main/java/alluxio/util/LogUtils.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.util;
+
+import alluxio.wire.LogInfo;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.commons.logging.impl.Jdk14Logger;
+import org.apache.commons.logging.impl.Log4JLogger;
+import org.apache.log4j.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.impl.Log4jLoggerAdapter;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Utility methods for working with log.
+ */
+@ThreadSafe
+public final class LogUtils {
+
+  private LogUtils() {} // prevent instantiation
+
+  /**
+   * Gets a logger's level with specify name, if the level argument is not null, it will set to
+   * specify level first.
+   * @param logName logger's name
+   * @param level logger's level
+   * @return an entity object about the log info
+   * @throws IOException if an I/O error occurs
+   */
+  public static LogInfo setLogLevel(String logName, String level) throws IOException {
+    LogInfo result = new LogInfo();
```

Comment:
Is returning a `LogInfo` with the name set to `null` a valid result? Add preconditions for assumptions made on the parameters passed in.